packages/lib-classifier/src/plugins/drawingTools/components/Mark/Mark.js

```diff
@@ -110,7 +115,14 @@ const Mark = forwardRef(function Mark(
     mark.x && mark.y
       ? `${transform} translate(${mark.x}, ${mark.y})`
       : transform
-  transform = mark.angle ? `${transform} rotate(${mark.angle})` : transform
+
+  if (mark.x_rotate && mark.y_rotate) {
+    transform = mark.angle
+      ? `${transform} rotate(${mark.angle}, ${mark.x_rotate}, ${mark.y_rotate})`
+      : transform
+  } else {
+    transform = mark.angle ? `${transform} rotate(${mark.angle})` : transform
+  }
```

Comment:
Maybe rewrite this to declare the transform string as a variable. It might make the code easier to read?
```suggestion
  if (mark.angle) {
    const rotateTransform = (mark.x_rotate && mark.y_rotate)
      ? `rotate(${mark.angle}), ${mark.x_rotate}, $mark.y_rotate})`
      : `rotate($mark.angle})`
    transform = `${transform} ${rotateTransform}`
  } 
```

Reply:
I like this! Much cleaner.